selfdrive/car/chrysler/values.py

```diff
@@ -22,23 +22,32 @@ class CAR:
 
   # Ram
   RAM_1500 = "RAM 1500 5TH GEN"
+  RAM_HD = "RAM HD 5TH GEN"
```

Comment:
does this cover both the 2500 and 3500? if so, any reason to merge them into one here?

Reply:
Yes this covers Ram 2500-5500. This is why I labeled it HD vs 2500